public/components/stub-modal/stub-modal.js

```diff
@@ -343,6 +343,19 @@ function StubModalInstanceCtrl($rootScope, $scope, $modalInstance, $window, conf
         }
     };
 
+    $scope.updateCommissionedLengthInComposer = function() {
+        const commissionedLength = $scope.stub.commissionedLength;
+        const missingCommissionedLengthReason = $scope.stub.missingCommissionedLengthReason;
+
+        [null, undefined].includes(commissionedLength)
+            ? wfComposerService.deleteFieldInPreviewAndLive(stub.composerId, 'commissionedLength')
+            : wfComposerService.updateFieldInPreviewAndLive(stub.composerId, 'commissionedLength', commissionedLength);
+
+        [null, undefined].includes(missingCommissionedLengthReason)
+            ? wfComposerService.deleteFieldInPreviewAndLive(stub.composerId, 'missingCommissionedLengthReason')
+            : wfComposerService.updateFieldInPreviewAndLive(stub.composerId, 'missingCommissionedLengthReason', missingCommissionedLengthReason);
```

Comment:
These methods will send post / delete requests to Composer even if no value has changed, there doesn't seem to be a straightforward way of comparing old and new fields in the scope of the form.